google/resource_bigquery_table.go

```diff
@@ -268,6 +285,17 @@ func resourceTable(d *schema.ResourceData, meta interface{}) (*bigquery.Table, e
 		table.TimePartitioning = expandTimePartitioning(v)
 	}
 
+	fields := d.Get("binding").(*schema.Set)
+	table.Schema = &bigquery.TableSchema{}
+
+	for i, v := range fields.List() {
+		field := v.(map[string]interface{})
+		table.Schema.Fields[i] = &bigquery.TableFieldSchema{
```

Comment:
This will segfault because table.Schema is nil at this point- take a look around the codebase for some examples of how to fill this in.

Reply:
I set `table.Schema` on line 289 – can it still be nil?

Reply:
Oh sorry, it's a different reason- the Fields array needs to be initialized: https://play.golang.org/p/2AsZv6v55tP is a stripped down version of what's happening here.

Reply:
Ah, thanks!